src/types/CanvasFile.ts

```diff
@@ -0,0 +1,51 @@
+interface CanvasFile {
+  // Canvas id for the file
+  id: number,
+  // Unique id for the file
+  uuid: string,
+  // Id of the folder containing the file
+  folder_id: number,
+  // Display name of the file
+  display_name: string,
+  // Name of the file
+  filename: string,
+  // Mime content type
+  'content-type': string,
```

Comment:
`'content-type'` -> `content_type`

Reply:
Unfortunately, this is set by Canvas so we can't rename :/